qa/rpc-tests/txoutsbyaddress.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+# Copyright (c) 2016 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+
+
+class TxOutsByAddressTest(BitcoinTestFramework):
+    """Tests -txoutsbyaddressindex"""
+
+    def __init__(self):
+        super().__init__()
+        self.setup_clean_chain = True
+        self.num_nodes = 3
+
+    def setup_network(self, split=False):
+        print("Setup network...")
+        self.nodes = []
+        self.nodes.append(start_node(0, self.options.tmpdir, ["-txoutsbyaddressindex"]))
+        self.nodes.append(start_node(1, self.options.tmpdir))
+        self.nodes.append(start_node(2, self.options.tmpdir))
+        self.is_network_split = False
+        self.sync_all()
+
+    def run_test(self):
+        print("Generating test blockchain...")
+
+        # Check that there's no UTXO on any of the nodes
+        for node in self.nodes:
+            assert_equal(len(node.listunspent()), 0)
+
+        # mining
+        connect_nodes(self.nodes[1], 0)
```

Comment:
Nit: you can move this up into `setup_network()`. I think all other tests do this